charts/harvester-cloud-provider/harvester-cloud-provider/0.1.300+up0.1.3/values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for harvester-cloud-provider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicasCount: 2
+
+image:
+  repository: rancher/harvester-cloud-provider
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```

Comment:
default tag should be set?

Reply:
ah I see, you use `{{ .Values.image.tag | default .Chart.AppVersion }}` above...

Can we not do that and instead just set the appVersion in the tag right here?

This is necessary since our current image-scanning process for generating rancher-images.txt relies on the values.yaml setting `*.repository` and `*.tag` somewhere nested in the values.yaml for it to be picked up... right now, this would be broken for airgap.

Reply:
same comment for `.Values.image.harvester.csiDriver.repository` in the csi driver